ui/src/app/components/gke-app-configuration-panels/create-gke-app.spec.tsx

```diff
@@ -104,190 +100,181 @@ describe(CreateGkeApp.name, () => {
   });
 
   describe.each([
-    [AppType.CROMWELL, defaultCromwellConfig, createListAppsCromwellResponse],
```

Comment:
git gets confused here about the diff - choosing Hide Whitespace will improve this.